deps/simdutf/simdutf.gyp

```diff
@@ -0,0 +1,35 @@
+{
+  'targets': [
+    {
+      'target_name': 'simdutf',
+      'type': 'static_library',
+      'include_dirs': ['.'],
+      'direct_dependent_settings': {
+        'include_dirs': ['.'],
+      },
+      'sources': ['simdutf.cpp'],
+      'cflags_cc': [ '-std=gnu++11', '-Wno-unused-command-line-argument' ],
+      'conditions': [
+        ['target_arch in "x64" and OS!="win"', {
+          'variables': {
+            'supports_vbmi2%': '<!(<!(echo ${CC_target:-${CC:-cc}}) -MM -E simdutf.cpp -mavx512vl -mavx512vbmi2 2>/dev/null || true)'
```

Comment:
Should these flags match the ones below?

Reply:
AFAIK These flags are sufficient to detect VBMI2 support

Reply:
I’m not sure how that relates to the question though … we test that `-mavx512vl` works but then pass `-mavx512f` instead of it, that seems like an odd mismatch?